docs/podman-events.1.md

```diff
@@ -11,7 +11,9 @@ podman\-events - Monitor Podman events
 Monitor and print events that occur in Podman. Each event will include a timestamp,
 a type, a status, name (if applicable), and image (if applicable).  The default logging
 mechanism is *journald*. This can be changed in libpod.conf by changing the `events_logger`
-value to `file`.  Only `file` and `journald` are the accepted.
+value to `file`.  Only `file` and `journald` are the accepted. A `none` logger is also
```

Comment:
s/the accepted/the accepted values/   or perhaps just accepted.

Reply:
Good catch, fixed